src/lazy/components/server.tsx

```diff
@@ -30,6 +33,11 @@ export function createComponentServer<C extends ComponentType<any>>({
     return (
       <>
         <input type="hidden" data-lazy-begin={dataLazyId} />
+        {defer !== PHASE.LAZY &&
+          SETTINGS.MANIFEST[moduleId] &&
+          SETTINGS.MANIFEST[moduleId].map(url => (
+            <link key={url} rel="preload" href={url} as="script" />
```

Comment:
Should `lazyAfterPaint` be `prefetch`?

Since you are inlining this in the output, should there also be a check to see if the `link` tag already exists like you have done in other logic? That would require an inline script though, which may be too slow.

Reply:
> Should lazyAfterPaint be prefetch?
We can make it like that. In reality was reading that `prefetch` is a bit discouraged and Chrome devs are suggesting to use `preload` anyway. So not sure about best thing to do here

> should there also be a check to see if the link tag already exists
Will do that for componentClient implementation, but on server feels impractical (plus the browser probably does not care much)